skops/io/_general.py

```diff
@@ -232,6 +233,25 @@ def object_get_instance(state, src):
     return instance
 
 
+def method_get_state(obj: Any, save_state: SaveState):
```

Comment:
Since it's not quite obvious, I would like to have a comment here that this is for methods bound to other instances, for future reference.